src/makeInfinityAdder.js

```diff
@@ -1,12 +1,35 @@
 'use strict';
 
 /***
- *
  * @return {function}
  */
-
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  function adder(value) {
+    if (arguments.length === 0) {
+      return sum;
```

Comment:
The sum should be reset to 0 after being returned to allow for subsequent independent calls to the adder function.